### DISCUSSION TITLE
Real time read_audio to file

### DISCUSSION THREAD

**Discussion by rodrigodifederico** [tayler6000/pyVoIP#171] [General]
Hi,

Does anyone have any suggestion about how to get the read_audio data and write it to a file in real time? Every time the caller says something, it saves the audio in a separated file.

The final goal is to transcript the audio using Vosk ( KaldiRecognizer ), so if anyone knows how to use the read_audio directly into KaldiRecognizer.AcceptWaveform, that would be awesome.

If there is any other easier way to transcribe in real time what the caller is saying, that would also be very appreciated.
Thank you!

**Comment by tyrohansen** [comment]
I need help on how to read audio into a file. so far what i have tried is not working
